spec/features/customers/show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'Customer show page', type: :feature do
+  before :each do
+    @supermarket = Supermarket.create!(name: 'Corner Market', location: '123 Food St.')
+    @customer1 = Customer.create!(name: 'Kaelin')
+    @item1 = Item.create!(name: 'Banana', price: 1, supermarket: @supermarket)
+    @item2 = Item.create!(name: 'Apple', price: 2, supermarket: @supermarket)
+    
+    @customer1.items << [@item1, @item2]
+  end
+
+  it 'shows the customer name and a list of their items with item details' do
+    visit customer_path(@customer1)
+
+    expect(page).to have_content(@customer1.name)
```

Comment:
I'd love to see more within blocks so we can really target what information is grouped with what. Here we're just testing that "Banana" and "$1" are on the page, but we have no way of knowing if those pieces of info are next to each other or not.